### ISSUE TITLE
Unable to Create Credentials at console.developers.google.com

### ISSUE THREAD

**Issue by FaserF** [Apipa169/Assistant-Relay-for-Hassio#48]
**Describe the bug**
I am unable to create credentials for this addon.
Sorry as this is no real issue with your addon itself, but I am stuck and dont know how to continue.
"To create an OAuth client ID, first provide a product name on the consent screen."
But at the consent screen I have filled everything. 

**To Reproduce**
1. Go to 'console.developers.google.com'
2. Click on 'credentials -> add new OAUTH credentials'
3. See error

**Expected behavior**
I want to know what to do :) 

**Screenshots**
![image](https://user-images.githubusercontent.com/12374134/95568470-8ddb6c00-0a24-11eb-94e1-17a5fdd7093b.png)
![image](https://user-images.githubusercontent.com/12374134/95568477-903dc600-0a24-11eb-97ec-fe2d54a210e8.png)

**System (please complete the following information):**
Hassio version: 0.116.1
System: Version 5.2 - Raspberry Pi 4

**Additional context**

**Comment by gtjamesa** [comment]
I'm having the same problem 

![](https://i.imgur.com/hcYDQxV.png)
![](https://i.imgur.com/AqVs048.png)

**Comment by Apipa169** [comment]
Hello,

Can you both try to follow these steps?
 https://greghesp.github.io/assistant-relay/docs/getting-started/configuration

**Comment by FaserF** [comment]
Hi. Thanks but I already tried following these steps.
I am unable to get to step 4 because of the mentioned error message above :/

**Comment by FaserF** [comment]
I was able to install your add-on as I had already set up google assistant manually with my home assistant and was able to use that project for your add-on too. But this is just a workaround. 
I tried setting up a new project 3 times with no success

**Comment by gtjamesa** [comment]
I solved this by doing the following:

1. Go to [Firebase Console](https://console.firebase.google.com/) and select your project
2. Settings (top-left) -> Project Settings
3. Ensure your support email is set and set the "Public-facing name" field on this page. I used "Assistant Relay Client"
4. Save
5. Go back and configure your OAuth consent screen. You'll either have to now fill in the information or it will be [completed](https://i.imgur.com/MjDL8GB.png). As I had previously submitted the form and it wasn't working, it had approved the fields I had entered before
6. Proceed with the remaining steps for creating OAuth client ID
